utils/stringUtils.js

```diff
@@ -31,6 +31,27 @@ const splitBenefitType = benefitType => {
     }
 
     return { description, code }
-}
+};
+
+const formatMobileNumber = number => {
+
+    let formattedNumber;
+
+    if (number.length > 11 && number.includes('+')) {
```

Comment:
Can you put the number 11 into a constant with an appropriate name such as `ELEVEN_DIGITS`.